[2024-04-24T16:48:54.770+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-24T16:48:54.782+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-24T16:48:50.005589+00:00 [queued]>
[2024-04-24T16:48:54.788+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-24T16:48:50.005589+00:00 [queued]>
[2024-04-24T16:48:54.788+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-24T16:48:54.802+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-24 16:48:50.005589+00:00
[2024-04-24T16:48:54.812+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-24T16:48:50.005589+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpqh_99sfk']
[2024-04-24T16:48:54.813+0000] {standard_task_runner.py:91} INFO - Job 285: Subtask get_files
[2024-04-24T16:48:54.816+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4162) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-24T16:48:54.817+0000] {standard_task_runner.py:63} INFO - Started process 4170 to run task
[2024-04-24T16:48:54.868+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files manual__2024-04-24T16:48:50.005589+00:00 [running]> on host 204d0a7369e0
[2024-04-24T16:48:55.157+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-24T16:48:50.005589+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-24T16:48:50.005589+00:00'
[2024-04-24T16:48:55.160+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-24T16:48:55.161+0000] {logging_mixin.py:188} INFO - 9 files found in /opt/***/dags/data
[2024-04-24T16:48:55.162+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/data/olist_orders_dataset.csv', '/opt/***/dags/data/olist_products_dataset.csv', '/opt/***/dags/data/olist_order_payments_dataset.csv', '/opt/***/dags/data/product_category_name_translation.csv', '/opt/***/dags/data/olist_sellers_dataset.csv', '/opt/***/dags/data/olist_order_reviews_dataset.csv', '/opt/***/dags/data/olist_order_items_dataset.csv', '/opt/***/dags/data/olist_customers_dataset.csv', '/opt/***/dags/data/olist_geolocation_dataset.csv']
[2024-04-24T16:48:55.162+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-24T16:48:55.203+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240424T164850, start_date=20240424T164854, end_date=20240424T164855
[2024-04-24T16:48:55.277+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-24T16:48:55.469+0000] {taskinstance.py:3482} INFO - 9 downstream tasks scheduled from follow-on schedule check
[2024-04-24T16:48:55.477+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
